### DISCUSSION TITLE
part-ii/2020-84/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#288] [Question comments]
Part II 2020 Linear Analysis: Paper 3, Section II, I 

https://questions.tripos.org/part-ii/2020-84/

**Comment by stevezshen** [comment]
<details>
<summary>Spoiler warning</summary>

(c) Let $S$ has rank $n$. Let $W = \text{im}\left(S^2-\lambda S\right)$. Then $S:H\to \mathbb{R}^n$ is a bijection on $W$. So $\forall h \in H\exists x\in W$ such that $S(h) = S(x)$. Then $h = x + y$ for some $y\in\ker(H)$. But $\ker(H)\leq \text{im}(S-\lambda I)$, so done. 

(d) $T-\lambda I$ is bounded below, let $\lVert T_n - \lambda I\rVert_{\text{op}}\to 0$, then $T_n - \lambda I$ is uniformly bounded below. Hence $\exists x_n$ such that $\lVert x_n\rVert\leq C$ for some $C$ and $\left(T_n-\lambda \right)x_n = h$. It follows that $\lVert h - (T-\lambda)x_n \rVert \to 0$ as $n\to\infty$. So the image is dense.

</details>
